portability-types-transfer/src/main/java/org/datatransferproject/types/transfer/models/blob/DtpDigitalDocument.java

```diff
@@ -0,0 +1,40 @@
+package org.datatransferproject.types.transfer.models.blob;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * This is intended to by a sub set of schema.org's DigitalDocumentWrapper
+ * (see: ).  Additional fields from the schema.org spec should
```

Comment:
nit: did you leave out a link?

Reply:
Sure did, added: https://schema.org/DigitalDocument